components/home/NotificationSection.tsx

```
@@ -1,12 +1,28 @@
 import React from "react";
 import { StyleSheet, View } from "react-native";
+import { useNoticePostStore } from "@/store/noticePostStore";
+import NotificationList from "@/components/NotificationList";
 
 interface NotificationSectionProps {}
 
 function NotificationSection({}: NotificationSectionProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)